README.md

```diff
@@ -246,7 +246,7 @@ GRIST_ADAPT_DOMAIN  | set to "true" to support multiple base domains (careful, h
 GRIST_APP_ROOT      | directory containing Grist sandbox and assets (specifically the sandbox and static subdirectories).
 GRIST_BACKUP_DELAY_SECS | wait this long after a doc change before making a backup
 GRIST_BOOT_KEY | if set, offer diagnostics at /boot/GRIST_BOOT_KEY
-GRIST_DATA_DIR      | directory in which to store document caches.
+GRIST_DATA_DIR      | directory in which to store documents. Defaults to `docs/` relative to the Grist application directory. A common value is `/persist/docs`, so that it will persist in a container.
```

Comment:
The sentence with `so that it will persist in a container` is useful, but may need further clarification itself. Don't know if you can characterize this as a common value? but rather the default setting in Grist containers maybe? In the Grist container, a set of resources are laid out on the assumption that users will mount a volume at `/persist`. Outside containers `/persist/...` isn't used at all I expect.

Reply:
It's a difficult to be art to be both clear and concise.

How about, "In Grist's default Docker image, its default value is `/persist/docs` so that it will be used as a mounted value."

Reply:
That sounds good @jordigh